src/docker-utils.coffee

```diff
@@ -218,7 +240,15 @@ do ->
 			.then ([ img ]) ->
 				knex('image').insert({ repoTag }) if !img?
 			.then ->
-				dockerProgress.pull(repoTag, onProgress)
+				Promise.join utils.getConfig('apiKey'), utils.getConfig('uuid'), (apiKey, uuid) ->
```

Comment:
This is part of the "sideload" API, usually used for external images, and it will be changed soon. I think there's no need to add credentials to this.